analyze.py

```diff
@@ -60,12 +66,21 @@ def run(args_dict):
     df = df[(df.date>=args_dict['date'][0]) & (df.date<=args_dict['date'][1])]
 
     vals = df.groupby(['date', 'project']).apply(lambda x: (x['end'] - x['start']).sum())
```

Comment:
Yup. The values for `start` and `end` when using sqlite come back as unicode, not Numpy timedelta's as with mysql. This shouldn't be too hard to fix up.